### ISSUE TITLE
v1/List not supported

### ISSUE THREAD

**Issue by timonbimon** [garden-io/garden#905]
## Bug

This issue comes out of a discussion in #814 
The resource type `v1/List` does not seem to be supported by garden currently.

```
Failed deploying service seldon-core-operator (from module seldon-core-operator). Here is the output:
————————————————————————————————————————————————————————————————————————————————
Unrecognized resource type v1/List
————————————————————————————————————————————————————————————————————————————————


1 deploy task(s) failed!
```

### Your environment

garden version: 0.10.0-1

**Comment by edvald** [comment]
Ah. I believed this had been fixed, but we may have missed a case or regressed back. I'll take a look.

**Comment by edvald** [comment]
I'm seeing a different issue when I try to deploy `seldon-core-operator`: 
```
CustomResourceDefinition.apiextensions.k8s.io "seldondeployments.machinelearning.seldon.io" is invalid: 
...
...properties[matchLabels].additionalProperties: Forbidden: additionalProperties cannot be set to false
...
```

**Comment by edvald** [comment]
I'd need to investigate further, but that particular issue doesn't _seem_ Garden related, but I won't rule it out yet. I'll take a closer look tomorrow, and flag this for our next patch release, which will no doubt happen quickly.

**Comment by timonbimon** [comment]
hmm that's weird.
here is the minimum example I need to reproduce the bug (needed to reset my Kubernetes cluster on Docker for Mac to make sure there is nothing else left from before)

garden.yml

```
project:
  name: test
  environments:
    - name: local
      providers:
        - name: local-kubernetes
          setupIngressController: false
```

seldon-core-operator/garden.yml
```
module:
  description: Seldon Core operator running Seldon Core CRD and Controller
  name: seldon-core-operator
  type: helm
  repo: https://storage.googleapis.com/seldon-charts
  chart: seldon-core-operator
  version: 0.3.0
  values:
    ambassador:
      enabled: true
      singleNamespace: true
    usageMetrics:
      enabled: true
```

**Comment by timonbimon** [comment]
![image](https://user-images.githubusercontent.com/14793026/60285802-992d8580-990e-11e9-9956-6c502308b917.png)

**Comment by edvald** [comment]
This is an odd one, having a hard time reproducing the same thing you're seeing. Keep getting that same thing I posted above, even after wiping my K8s cluster. Could you try the new v0.10.0 release and see if you get the same error I do?

**Comment by timonbimon** [comment]
hmm yea weird, I still get the same error when on 0.10.0

I tried on a friends computer and he gets the same error that you do. However you can install the seldon-core-operator using helm without a problem, so I feel it might still be garden related...

**Comment by eysi09** [comment]
I can reproduce so that's a step forward. Will report back once I've figured this out.

**Comment by eysi09** [comment]
Hi @timonbimon, this was fixed in #934 and will be included in our next release. Hopefully later today.
